airflow_prometheus_exporter/prometheus_exporter.py

```diff
@@ -277,6 +278,57 @@ def get_task_duration_info():
         )
 
 
+def get_landing_times():
+    """Duration of successful tasks in seconds."""
+    with session_scope(Session) as session:
+        max_execution_dt_query = (
+            session.query(
+                DagRun.dag_id,
+                func.max(DagRun.execution_date).label("max_execution_dt"),
+            )
+            .join(DagModel, DagModel.dag_id == DagRun.dag_id,)
+            .filter(
+                DagModel.is_active == True,  # noqa
+                DagModel.is_paused == False,
+                DagRun.state == State.SUCCESS,
+                DagRun.end_date.isnot(None),
+            )
+            .group_by(DagRun.dag_id)
+            .subquery()
+        )
+
+        return (
+            session.query(
+                TaskInstance.dag_id,
+                TaskInstance.task_id,
+                TaskInstance.start_date,
+                TaskInstance.end_date,
+                TaskInstance.execution_date,
+                DagModel.schedule_interval,
+            )
+            .join(
+                max_execution_dt_query,
+                and_(
+                    (TaskInstance.dag_id == max_execution_dt_query.c.dag_id),
+                    (
+                        TaskInstance.execution_date
+                        == max_execution_dt_query.c.max_execution_dt
+                    ),
+                ),
+            )
+            .join(
+                 DagModel,
```

Comment:
Why are you joining with DagModel twice? Wouldn't it make more sense to add the schedule interval in the previous subquery?